<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1197</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CNPJ Finder v1.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>cnpj_finder.ico</normaloff>cnpj_finder.ico</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="cnpj_fields">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>40</y>
      <width>401</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_input_cnpj">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>13</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Digite o CNPJ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_input">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>13</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::InputMethodHint::ImhPreferNumbers</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_cnpj_formated">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>53</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CNPJ formatado</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_cp_astrk">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>93</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Copiar</string>
     </property>
     <property name="icon">
      <iconset theme="edit-copy"/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_output_dots">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>53</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_cnpj_formated_2">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>100</y>
       <width>111</width>
       <height>10</height>
      </rect>
     </property>
     <property name="text">
      <string>CNPJ formatado(*)</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_cp_dots">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>53</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Copiar</string>
     </property>
     <property name="icon">
      <iconset theme="edit-copy"/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cls_input">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>13</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Limpar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_output_astrk">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>93</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_location">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>230</y>
      <width>501</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
      <underline>false</underline>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Localização</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_number">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>197</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Número:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_output_city">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>157</y>
       <width>281</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_output_neighborhood">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>117</y>
       <width>251</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_city">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>157</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Município</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_complement">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>227</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Complemento</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_output_complement">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>227</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_street">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>77</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Logradouro</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_neighborhood">
     <property name="geometry">
      <rect>
       <x>67</x>
       <y>118</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bairro</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_output_cep">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>40</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_uf">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>197</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>UF:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_output_street">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>77</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_cep">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>41</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CEP</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_cp_cep">
     <property name="geometry">
      <rect>
       <x>291</x>
       <y>40</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Copiar</string>
     </property>
     <property name="icon">
      <iconset theme="edit-copy"/>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_cp_street">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>77</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Copiar</string>
     </property>
     <property name="icon">
      <iconset theme="edit-copy"/>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_cp_neighborhood">
     <property name="geometry">
      <rect>
       <x>381</x>
       <y>117</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Copiar</string>
     </property>
     <property name="icon">
      <iconset theme="edit-copy"/>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_cp_city">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>158</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Copiar</string>
     </property>
     <property name="icon">
      <iconset theme="edit-copy"/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_contact">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>230</y>
      <width>520</width>
      <height>111</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Contato</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_phone">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Telefone</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_email">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>34</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>E-mail</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_output_email">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>33</y>
       <width>361</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_output_phone">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_cp_email">
     <property name="geometry">
      <rect>
       <x>451</x>
       <y>33</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Copiar</string>
     </property>
     <property name="icon">
      <iconset theme="edit-copy"/>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_cp_phone">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>70</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Copiar</string>
     </property>
     <property name="icon">
      <iconset theme="edit-copy"/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="company_informations">
    <property name="geometry">
     <rect>
      <x>431</x>
      <y>20</y>
      <width>741</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Dados da empresa</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <widget class="QLabel" name="label_status">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Situação:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_output_social_name">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>551</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_fake_name">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>81</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome fantasia</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_output_fake_name">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>551</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_last_update">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Última atualização:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_type">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_social_name">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>41</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Razão social</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_status_2">
     <property name="geometry">
      <rect>
       <x>251</x>
       <y>160</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_size">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>160</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Porte:</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_cp_social_name">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>40</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Copiar</string>
     </property>
     <property name="icon">
      <iconset theme="edit-copy"/>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_cp_fake_name">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>80</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Copiar</string>
     </property>
     <property name="icon">
      <iconset theme="edit-copy"/>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView_qsa">
    <property name="geometry">
     <rect>
      <x>548</x>
      <y>380</y>
      <width>621</width>
      <height>251</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>205</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>57</number>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_qsa">
    <property name="geometry">
     <rect>
      <x>551</x>
      <y>360</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>QSA</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1197</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfigura_es">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <widget class="QMenu" name="menuTema">
     <property name="title">
      <string>Preferências</string>
     </property>
     <widget class="QMenu" name="menuTema_2">
      <property name="title">
       <string>Tema</string>
      </property>
      <addaction name="actionWhite_2"/>
      <addaction name="actionDark_2"/>
     </widget>
     <addaction name="menuTema_2"/>
    </widget>
    <addaction name="menuTema"/>
    <addaction name="actionHist_rico"/>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menuConfigura_es"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::RightToLeft</enum>
   </property>
  </widget>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionWhite_2">
   <property name="text">
    <string>White</string>
   </property>
  </action>
  <action name="actionDark_2">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionHist_rico">
   <property name="text">
    <string>Histórico</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
